repl/lib/js/main.js

```diff
@@ -1,3 +1,6 @@
+import sanctuary from 'sanctuary';
+import ramdaFantasy from 'ramda-fantasy';
```

Comment:
if we're to do this, i think we need to update package.json to refer to the "latest" for both libs.